pkg/manifests/config.go

```diff
@@ -96,8 +96,7 @@ type K8sPrometheusAdapter struct {
 }
 
 type EtcdConfig struct {
-	Enabled    *bool  `json:"enabled"`
-	ServerName string `json:"serverName"`
+	Enabled *bool `json:"-"`
```

Comment:
Is this still necessary, given that `EtcdConfig` in `Config` is disabled anyways?

Reply:
I intentionally left this to modify the least amount of code, but also seems like a reasonable strategy to me. Essentially what's happening here is that when the right certificates are present in the `Secret`, then this field gets set to enabled.